src/hooks/useAiAgent.ts

```diff
@@ -109,7 +116,14 @@ export interface AiHookResponse {
   complete: boolean;
 }
 
-export function useAiHook(integrationIds: Array<string>, aiQuery: boolean, profileId?: string): AiHookResponse {
+export function useAiHook(
```

Comment:
nit: this public function needs some docs

Reply:
Done.